### PR TITLE
[Issue5] abstract room

### PR DESCRIPTION
**Pull request by patrickv83**

### CODE REVIEW

**Review by jalyxis55 — CHANGES_REQUESTED** [review]
Room() doesn't need (x,y). I think an (x,y) will be used with player movement rather than where the room sits in the room.

**Review by jalyxis55 — DISMISSED** [review]
I lied, room does have (x, y). sorry.

**Comment by patrickv83** [comment]
Coordinates are back in Room class, stub for EmptyRoom class added

File: demonata/room.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    @abstractmethod
+    def generateDescription(self):
+        self.description = "Nothing to see here"
+
+    @abstractmethod
+    def getDescription(self):
+        print self.description
+
+
+class EmptyRoom(Room):
```
Comment (claudecyrill) [inline demonata/room.py:26]: Would need to get scrapped on merge since empty room is defined the latest pull request.

**Review by claudecyrill — APPROVED** [review]

**Pull request merged**